sdk/iOS/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -344,4 +347,10 @@ - (void)playPauseFromAdTappedNotification {
 - (void)onApplicationDidBecomeActive:(NSNotification *)notification {
   MACaptionAppearanceSetDisplayType(kMACaptionAppearanceDomainUser, kMACaptionAppearanceDisplayTypeForcedOnly);
 }
+
+- (void)notifyFullScreenChange:(BOOL) isFullScreen {
+    [[NSNotificationCenter defaultCenter] postNotificationName:OOOoyalaPlayerFullScreenChangedNotification object:self
+        userInfo:[[NSDictionary alloc] initWithObjectsAndKeys:@(isFullScreen), @"fullScreen", nil]];
```

Comment:
So, I know you're new to Obj-C but could you change the NSDictionary initialization? There's a shorter syntax, let me show you:
```
// Long old initialization
[[NSDictionary alloc] initWithObjectsAndKeys:@(isFullScreen), @"fullScreen", nil];
// new good looking initialization :)
@{@"fullscreen": @(isFullscreen)};
```